It seems that flexdll-linked bytecode cannot survive a fork().

--- origsrc/cameleon-1.9.19/configure.in	2009-09-24 06:46:38.000000000 -0500
+++ src/cameleon-1.9.19/configure.in	2010-02-14 13:19:29.902815500 -0600
@@ -33,7 +33,8 @@ ROOT=`pwd`
 
 # Check for Ocaml compilers
 
-ocaml unix.cma str.cma utils/checkocaml.ml ocaml_config.sh || exit 1
+ocamlopt -o utils/checkocaml unix.cmxa str.cmxa utils/checkocaml.ml || exit 1
+./utils/checkocaml ocaml_config.sh || exit 1
 . ./ocaml_config.sh
 
 echo "ocaml library path is $OCAMLLIB"
